<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicForm</class>
 <widget class="QWidget" name="BasicForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCommandLinkButton" name="MyProfileLinkButton">
       <property name="text">
        <string>My Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="MyCoursesLinkButton">
       <property name="text">
        <string>My Courses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="MyGroupLinkButton">
       <property name="text">
        <string>My Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="InformationLinkButton">
       <property name="text">
        <string>Information</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="PhotoLabel">
           <property name="text">
            <string>Photo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="ProfilePhoto">
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="ProfoleSurnamelabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Surname:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileNamelabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileFacultylabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Faculty:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileGroupLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Group:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileScoreLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Total score:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileArrearsLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Arrears:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="ProfileSurName">
             <property name="text">
              <string>ProfileSurName</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfielName">
             <property name="text">
              <string>ProfileName</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileFaculty">
             <property name="text">
              <string>ProfileFaculty</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileGroup">
             <property name="text">
              <string>ProfileGroup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileTotalScore">
             <property name="text">
              <string>ProfileTotalScore</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProfileArrears">
             <property name="text">
              <string>ProfileArrears</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView">
         <property name="tabKeyNavigation">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>37</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>190</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>31</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>37</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="infolable">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sections and improvements will appear with updates</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
